scripts/templates/component/stories/Component.stories.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from "react";
 import { storiesOf } from "@storybook/react";
 import { withReadme } from "storybook-readme";
-import ${Component} from "../${Component}";
+import { ${Component} } from "../../index";
```

Comment:
Should be `"../index"`

Reply:
Why should be `../index`? the reason we are using `../../index` is to target the root path as the projects will.

Reply:
Ha! I get it now, so after scaffolding one should put the component into the main index

Reply:
Correct. I added that to the description too.

Thanks.